src/app/cli/src/client.ml

```diff
@@ -3,6 +3,9 @@ open Async
 open Signature_lib
 open Coda_base
 
+let print_rpc_error error =
+  eprintf !"Encountered RPC connection error: %s\n" (Error.to_string_hum error)
```

Comment:
I'd say `Encountered` isn't necessary.

Since there's no sexp here, the `!` isn't needed.